tests/rptest/services/admin.py

```diff
@@ -223,6 +223,16 @@ def mark_transaction_expired(self,
         path = f"partitions/{namespace}/{topic}/{partition}/mark_transaction_expired?id={pid['id']}&epoch={pid['epoch']}"
         return self._request("post", path, node=node)
 
+    def delete_partition_from_transaction(self,
+                                          tid,
+                                          partition_info,
+                                          node=None):
+        """
+        Delete partition from transaction
+        """
+        path = f"transaction/{tid}/delete_partition?namespace={partition_info['ns']}&topic={partition_info['topic']}&partition_id={int(partition_info['partition_id'])}&etag={partition_info['etag']}"
```

Comment:
nit: you can build the query parameters with something like `"&".join([f"{k}={v}" for k,v in d.items()])` where `d` contains the query parameters. also mentioned adding the ntp to the URL path itself as another option that seems more common for our APIs?

Reply:
can you split up the construction of the URL so that it isn't so wide?